tests/tests.bats

```diff
@@ -0,0 +1,182 @@
+# shellcheck disable=SC2148
+# shellcheck disable=SC2329
+setup() {
+  load '/bats-libraries/bats-support/load.bash'
+  load '/bats-libraries/bats-assert/load.bash'
+  load '/bats-libraries/bats-file/load.bash'
+}
+
+# Use a name that upon cluster inspection reveals that this
+# Helm chart release was installed/managed by this test suite.
+export TEST_HELM_RELEASE_NAME="nvidia-dra-driver-gpu-batssuite"
+
+# Note(JP): bats swallows output of setup upon success (regardless of cmdline
+# args such as `--show-output-of-passing-tests`). Ref:
+# https://github.com/bats-core/bats-core/issues/540#issuecomment-1013521656 --
+# it however does emit output upon failure.
+# shellcheck disable=SC2329
+setup_file() {
+  # Create Helm repo cache dir and point `helm` to it, otherwise `Error:
+  # INSTALLATION FAILED: mkdir /.cache: permission denied`
+  HELM_REPOSITORY_CACHE=$(mktemp -d -t helm-XXXXX)
+  export HELM_REPOSITORY_CACHE
+
+  # Consumed by the helm CLI.
+  export HELM_REPOSITORY_CONFIG=${HELM_REPOSITORY_CACHE}/repo.cfg
+  export CRD_UPGRADE_URL="https://raw.githubusercontent.com/NVIDIA/k8s-dra-driver-gpu/$TEST_CRD_UPGRADE_TARGET_GIT_REF/deployments/helm/nvidia-dra-driver-gpu/crds/resource.nvidia.com_computedomains.yaml"
+
+  # Prepare for allowing to run this test suite against k8s with host-provided
+  # GPU driver.
+  export TEST_NVIDIA_DRIVER_ROOT="/run/nvidia/driver"
```

Comment:
maybe expose via makefile target

Reply:
Yeah, that seems reasonable -- especially once we encapsulate it all

Reply:
exposed in makefile now